domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncerWorker.cs

```diff
@@ -81,6 +82,22 @@ private async Task PerformPeriodicTask(CancellationToken stoppingToken)
         }
     }
 
+    private async Task Sleep(CancellationToken cancellationToken)
+    {
+        if (options.SleepType == MeasurementsSyncerSleepType.Hourly)
```

Comment:
Why these different sleeptypes?

Reply:
Only breaks if we don't set it. We need it to run the MeasurementsSyncer faster for integration tests